tests/main/uc20-create-partitions-encrypt/task.yaml

```diff
@@ -82,7 +82,7 @@ execute: |
         ./gadget-dir "$LOOP"
 
     echo "Check that the key file was created"
-    test "$(stat --printf=%s ./keyfile)" = 1095
+    test "$(stat --printf=%s ./keyfile)" -ge 1000
```

Comment:
What's the idea here?

Reply:
The size of the sealed key is still unstable and I'm testing against a reasonable minimum size.